### PR TITLE
Update address correction page to use new strings

### LINKED ISSUE #6639: Update text on the address confirmation page
During a discussion with Shreya and Sijia about how the address confirmation page interacts with the Previous/Review buttons, we decided to update the text on that page to be a bit more friendly. Here's the new text that we should use (text verified by https://hemingwayapp.com/ to be a 5th grade reading level):

Title: Confirm your address

Body:
We were not able to find your address.

Please check your address for spelling or choose an option from the list provided. 

Headings:
Address you entered:

Suggested addresses:

Button text at the bottom if came from "Previous" or "Review" button clicks
Confirm address

### PR DESCRIPTION
**Pull request by caitlinshk**
### Description

This PR updates the text on the address correction page to use the new strings specified by Sijia.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Created unit and/or browser tests which fail without the change (if possible) -- **existing ones are sufficient**
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)

#### User visible changes

- [x] Followed steps to [internationalize new strings](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#internationalization-for-application-strings)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag) -- **Strings have been translated**
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order

### Instructions for manual testing

- Fill out an address question that has address correction labeled with "Legit Address" as the street name. On the address correction page, verify the new strings are shown and the suggestions are shown.
- Fill out an address question that has address correction labeled with "Bogus Address" as the street name. On the address correction page, verify the new strings are shown and no suggestions are shown.

### Issue(s) this completes

Fixes #6639

### CODE REVIEW

**Comment by caitlinshk** [comment]
* **#7059** <a href="https://app.graphite.dev/github/pr/civiform/civiform/7059?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @caitlinshk and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: browser-test/src/support/applicant_questions.ts [hunk 1]
```diff
@@ -519,9 +519,20 @@ export class ApplicantQuestions {
...
-    expect(await this.page.innerText('h2')).toContain(header)
+  async expectVerifyAddressPage(hasAddressSuggestions: boolean) {
+    expect(await this.page.innerText('h2')).toContain('Confirm your address')
+    if (hasAddressSuggestions) {
+      // Note: If there's only one suggestion, the heading will be "Suggested address"
+      // not "Suggested addresses". But, our browser setup always returns multiple
+      // suggestions so we can safely assert the heading is always "Suggested addresses".
+      await expect(
+        this.page.getByRole('heading', {name: 'Suggested addresses'}),
+      ).toBeVisible()
```
Comment (greg-hyde) [inline browser-test/src/support/applicant_questions.ts:530]: optional: You should be able to pass a visible boolean to toBeVisible so you could do await expect(blah).toBeVisible(hasAddressSuggestions) https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-visible so you don't need this if/else

**Review by greg-hyde — APPROVED** [review]

File: browser-test/src/support/applicant_questions.ts (hunk 1 above)
Comment (caitlinshk) [reply browser-test/src/support/applicant_questions.ts]: Ooh nice, updated.

File: browser-test/src/support/applicant_questions.ts [hunk 2]
```diff
@@ -519,9 +519,14 @@ export class ApplicantQuestions {
     ).toEqual(0)
   }
 
-  async expectVerifyAddressPage(validAddress: boolean) {
-    const header = validAddress ? 'Verify address' : 'No valid address found'
-    expect(await this.page.innerText('h2')).toContain(header)
+  async expectVerifyAddressPage(hasAddressSuggestions: boolean) {
+    expect(await this.page.innerText('h2')).toContain('Confirm your address')
+    // Note: If there's only one suggestion, the heading will be "Suggested address"
```
Comment (rockycodes) [inline browser-test/src/support/applicant_questions.ts:524]: praise: helpful note!

**Review by rockycodes — COMMENTED** [review]
looks good to me!

**Pull request merged**